### PR TITLE
Theme factoring

### PR DESCRIPTION
**Pull request by spryor**
# Type of change? 

- [ ] Bug fix
- [x] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Contains a first pass at the light theme and some sass refactoring to that effect

### CODE REVIEW

File: src/components/shared/forms/styles/formControl.scss [hunk 1]
```diff
@@ -52,25 +52,25 @@ $textareaHeight: 60px;
...
       }
 
-      &::-webkit-input-placeholder { color: themed('colorLightestSecond'); }
-          &:-ms-input-placeholder { color: themed('colorLightestSecond'); }
-              &::-moz-placeholder { color: themed('colorLightestSecond'); }
-                &:-moz-placeholder { color: themed('colorLightestSecond'); }
+      &::-webkit-input-placeholder { color: themed('colorInputPlaceholderText'); }
+          &:-ms-input-placeholder { color: themed('colorInputPlaceholderText'); }
+              &::-moz-placeholder { color: themed('colorInputPlaceholderText'); }
+                &:-moz-placeholder { color: themed('colorInputPlaceholderText'); }
```
Comment (ppathan) [inline src/components/shared/forms/styles/formControl.scss]: nit: The tabs don't seem consistent is that expected?

File: src/components/shared/forms/styles/formControl.scss [hunk 2]
```diff
@@ -25,8 +25,8 @@ $textareaHeight: 60px;
     &:disabled {
       -webkit-touch-callout: none;
         -webkit-user-select: none;
-        -khtml-user-select: none;
-          -moz-user-select: none;
+         -khtml-user-select: none;
+           -moz-user-select: none;
```
Comment (ppathan) [inline src/components/shared/forms/styles/formControl.scss]: is it the style to leave one extra space in each line?

File: src/components/shared/forms/styles/formControl.scss [hunk 3]
```diff
@@ -52,25 +52,25 @@ $textareaHeight: 60px;
...
-                &::-moz-placeholder { color: themed('colorMidThird'); }
-                  &:-moz-placeholder { color: themed('colorMidThird'); }
+        background-color: themed('colorDisabledInputBg');
+        color: themed('colorDisabledInputBg');
+        border: 1px solid themed('colorDisabledInputBg');
+
+        &::-webkit-input-placeholder { color: themed('colorDisabledInputBg'); }
+            &:-ms-input-placeholder { color: themed('colorDisabledInputBg'); }
+                &::-moz-placeholder { color: themed('colorDisabledInputBg'); }
+                  &:-moz-placeholder { color: themed('colorDisabledInputBg'); }
```
Comment (ppathan) [inline src/components/shared/forms/styles/formControl.scss]: nit: same comment about tabs

File: src/styles/_themes.scss [hunk 4]
```diff
@@ -73,25 +68,210 @@ $themes: (
...
+    colorGridRowBackground: $colorWhite,
+    colorGridCellText: #212121,
+    colorGridRowHoverBackground: #dadada,
+    colorGridSelectedRowBorder: $colorSystem,
+    colorGridHeaderText: #767676,
+
+    colorCellRendererSvgFill: #212121,
+    colorCellRendererSvgFillHighlight: #212121,
+    colorCellRendererText: #212121,
+    colorCellRendererTextHighlight: #212121,
```
Comment (ppathan) [inline src/styles/_themes.scss]: The light theme has values as hard coded compared to dark theme that has variables. Is there a TODO to change later?

File: src/components/shared/forms/styles/formControl.scss (hunk 2 above)
Comment (spryor) [reply src/components/shared/forms/styles/formControl.scss]: nope, fixed

File: src/components/shared/forms/styles/formControl.scss (hunk 1 above)
Comment (spryor) [reply src/components/shared/forms/styles/formControl.scss]: nope, this was left over from simulation, fixed

File: src/components/shared/forms/styles/formControl.scss (hunk 3 above)
Comment (spryor) [reply src/components/shared/forms/styles/formControl.scss]: fixed

File: src/styles/_themes.scss (hunk 4 above)
Comment (spryor) [reply src/styles/_themes.scss]: There is a TODO now

**Pull request merged**